tasks.gradle

```diff
@@ -1,6 +1,7 @@
 task javadocs(type: Javadoc) {
     source = android.sourceSets.main.java.source
     classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
+    excludes = ['**/*.kt']
```

Comment:
Why do we exclude kt files from the docs?

Reply:
Margarita was getting an error trying to deploy locally without this, but I just checked and I successfully deployed locally so I removed this line